src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/ProjectServices/VsManagedLanguagesProjectSystemServices.cs

```diff
@@ -58,10 +58,11 @@ internal class VsManagedLanguagesProjectSystemServices
 
         static VsManagedLanguagesProjectSystemServices()
         {
-            _referenceMetadata = Array.CreateInstance(typeof(string), 3);
+            _referenceMetadata = Array.CreateInstance(typeof(string), 4);
```

Comment:
i know this isn't related, but can we please simplify this to :
_referenceMetadata = new string[] {..... }  ?